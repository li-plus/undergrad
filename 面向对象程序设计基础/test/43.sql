create database oop;
use oop;
create table a(a int); 
create table b(a int); 
create table c(a int); 
create table d(a int); 
show tables; 
/********************** Sample Output *************************
Tables_in_oop
a
b
c
d
**************************************************************/
drop table a, c, d; 
show tables; 
/********************** Sample Output *************************
Tables_in_oop
b
**************************************************************/
drop database oop; 