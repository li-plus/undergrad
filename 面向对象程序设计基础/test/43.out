Tables_in_oop
a
b
c
d
Tables_in_oop
b